### PR TITLE
Bug fix: bastion regen is always disabled

### PR DESCRIPTION
**Pull request by Aleksey-Terzi**

### CODE REVIEW

**Comment by DevotedServant** [comment]
Can one of the admins verify this patch? @ProgrammerDan @Maxopoly

"ok to test" will build any commits made to this PR
"test this please" will build this PR once
"retest this please" will make another single build for this PR
"add to whitelist" will whitelist the creator of this PR and make jenkins automatically build any PRs made by him

File: src/main/java/isaac/bastion/BastionType.java [hunk 1]
```diff
@@ -86,6 +86,8 @@ public BastionType(String name, MaterialData material, String itemName, List<Str
 		this.explodeOnBlock = explodeOnBlock;
 		this.explodeOnBlockStrength = explodeOnBlockStrength;
 		this.damageFirstBastion = damageFirstBastion;
+		this.regenTime = regenTime;
```
Comment (ProgrammerDan) [inline src/main/java/isaac/bastion/BastionType.java:89]: how in the world did I miss this line

**Comment by ProgrammerDan** [comment]
Dang, good catches. I'm going to need to backport this to 1.10 until we upgrade, sigh. Thanks Aleks!

**Comment by ProgrammerDan** [comment]
ok to test

**Pull request merged**